adding class 4 and dog excercise

bahaadabbagh opened the pull request:
I think I'm up to date now.

louisrli commented:
a few fixes are needed:
1. dog exercise shouldn't be added here, it should be in your own repo on your page, so delete it
2. don't put spaces in filenames (this is by convention, usually we use `-`). please change the filenames to be without spaces.
3. you put the files in `class 4` instead of `class4` so they are in the wrong directory :) this needs to be fixed

louisrli commented:
bahaa, can you update this?

bahaadabbagh commented:
Hi Louis,

Sorry for the late reply, I just saw your email.
Must have slipped through the notifications of slack emails.
So I fixed the names but dog exercise wasn't deleted when I deleted it from
my local repo and committed it.
Any way to delete it online? I already deleted it locally.



On Thu, Apr 2, 2020 at 1:27 AM Louis Li <notifications@github.com> wrote:

> bahaa, can you update this?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ReCoded-Org/bootcamp-istanbul-2020mar/pull/61#issuecomment-607520832>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AO34LI76GIJ6KTIMAE5HYOLRKO5UZANCNFSM4LXOV25Q>
> .
>

louisrli commented:
1. your folder name still has a space :)
https://github.com/ReCoded-Org/bootcamp-istanbul-2020mar/pull/61/files
if you look at the folder names here
2. try `git rm -rf dogexercise` from the top directory of the git repository (or whatever exercise was called). lmk if that works

bahaadabbagh commented:
Hi louis,
I think I fixed it now. I added class6 and class4. what was missing was I wasn't fetching the files and that created a conflict. can you check now if it's correct?

louisrli commented:
nicely done, very clean. lgtm
